src/main/java/uk/gov/ch/model/update/OverseasEntityManagingOfficerData.java

```diff
@@ -0,0 +1,138 @@
+package uk.gov.ch.model.update;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import uk.gov.companieshouse.api.model.common.Address;
+
+@Entity
+public class OverseasEntityManagingOfficerData {
+
+    @Id
+    @Column(name = "MO_ID")
```

Comment:
There are 2 addresses for a managing officer and it's not clear which we are retrieving here. Either add a JSON level for residential/usual address (my preference) or prefix the address fields e.g. "residentialXyz"

Reply:
Sorry, was confused by db and json being in same object. Maybe consider breaking this apprat.

Reply:
Should i not leave like this as it is done elsewhere the same way with many more fields?

Eg. ActiveDirectorDetails.java